src/styles/edgeAttributes.ts

```diff
@@ -50,7 +59,7 @@ export class EdgeAttributes extends ItemAttributes {
           color = rule.style.color;
         } else if (typeof rule.style.color === 'object') {
           const propValue = Tools.getIn(data, rule.style.color.input);
-          color = ItemAttributes.autoColor(`${propValue}`, rule.style.ignoreCase);
+          color = ItemAttributes.autoColor(`${propValue}`, rule.style.color.ignoreCase);
```

Comment:
That is a hidden bug as ignoreCase doesn't exist in style but in the color attribute

Reply:
we never had an error because before it was always considered as undefined when passing it to getRandomUniqueColorPaletteIndex